build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+SHA=`git rev-parse --short --verify HEAD`
+
+echo "\newcommand{\YellowPaperVersionNumber}{$SHA}" > Version.tex
+
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
+bibtex Paper
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
```

Comment:
I think these command should be concatenated with `&& \`.   Then, if one of the commands fail, Travis can notice.